NOnion.Tests/MonohopCircuits.cs

```diff
@@ -70,5 +73,49 @@ public async Task CanReceiveCompressedConsensusOverMonohopCircuit()
             Assert.That(response.Contains("network-status-version"));
         }
 
+        private async Task<ServerDescriptorEntry> GetRouter ()
+        {
+            using TorGuard guard = await TorGuard.NewClientAsync(torServer);
+            TorCircuit circuit = new(guard);
+            TorStream stream = new(circuit);
+
+            await circuit.CreateAsync(FSharpOption<CircuitNodeDetail>.None);
+            await stream.ConnectToDirectoryAsync();
+
+            var httpClient = new TorHttpClient(stream, torServer.Address.ToString());
+            var serverDescriptors = ServerDescriptorsDocument.Parse(await httpClient.GetAsStringAsync("/tor/server/all", false));
+
+            return serverDescriptors.Routers.First(x => FSharpOption<string>.get_IsSome(x.NTorOnionKey) && FSharpOption<string>.get_IsSome(x.Fingerprint) && FSharpOption<int>.get_IsSome(x.DirectoryPort) &&  x.DirectoryPort.Value != 0);
+        }
+
+        [Test]
+        public async Task CanReceiveCompressedConsensusOverNonFastMonohopCircuit()
+        {
+            ServerDescriptorEntry server = null;
+            try
+            {
+                server = await GetRouter();
+            }
+            catch
+            {
+                Assert.Inconclusive();
+            }
+
+            var fingerprintBytes = Hex.ToByteArray(server.Fingerprint.Value);
+            var nTorOnionKeyBytes = Base64Util.FromString(server.NTorOnionKey.Value);
+            var nodeDetail = new CircuitNodeDetail(nTorOnionKeyBytes, fingerprintBytes);
+
+            using TorGuard guard = await TorGuard.NewClientAsync(IPEndPoint.Parse($"{server.Address.Value}:{server.OnionRouterPort.Value}"));
+            TorCircuit circuit = new(guard);
+            TorStream stream = new(circuit);
+
+            await circuit.CreateAsync(nodeDetail);
+            await stream.ConnectToDirectoryAsync();
+
+            var httpClient = new TorHttpClient(stream, torServer.Address.ToString());
+            var response = await httpClient.GetAsStringAsync("/tor/status-vote/current/consensus", false);
+
+            Assert.That(response.Contains("network-status-version"));
```

Comment:
should be Assert.That(response, Is.StringContaining("bla")) or something, not the above

Reply:
This is already fluent enough and it also already exists in other tests, so no point in changing it now

Reply:
response.Contains() is an extension method; I hate extension methods! And the other syntax I propose goes more in line with the other asserts.

Reply:
We have the same Contains assert used in 2 other tests, so we have to either change all of them or let this one slide as well also Contains is not an extension method